src/istio/mixerclient/attribute_compressor.cc

```diff
@@ -159,7 +159,7 @@ class BatchCompressorImpl : public BatchCompressor {
 
 }  // namespace
 
-GlobalDictionary::GlobalDictionary() {
+GlobalDictionary::GlobalDictionary() : global_dict_(GetGlobalDictionary()) {
   const std::vector<std::string>& global_words = GetGlobalWords();
   for (unsigned int i = 0; i < global_words.size(); i++) {
     global_dict_[global_words[i]] = i;
```

Comment:
Delete these lines since index is already available?

Reply:
Hmm, my force push only picked up sign off. Will fix!